test/addons/olm/start

```diff
@@ -9,33 +9,17 @@ import sys
 import drenv
 from drenv import kubectl
 
-VERSION = "v0.27.0"
-BASE_URL = f"https://github.com/operator-framework/operator-lifecycle-manager/releases/download/{VERSION}"
 NAMESPACE = "olm"
 
 
 def deploy(cluster):
-    print("Deploying olm crds")
+    print("Deploying olm")
 
     # Using Server-side Apply to avoid this failure:
     #   The CustomResourceDefinition "clusterserviceversions.operators.coreos.com"
     #   is invalid: metadata.annotations: Too long: must have at most 262144 bytes
     # See https://medium.com/pareture/kubectl-install-crd-failed-annotations-too-long-2ebc91b40c7d
-    kubectl.apply(
-        f"--filename={BASE_URL}/crds.yaml",
-        "--server-side=true",
-        context=cluster,
-    )
-
-    print("Waiting until cdrs are established")
-    kubectl.wait(
-        "--for=condition=established",
-        f"--filename={BASE_URL}/crds.yaml",
-        context=cluster,
-    )
```

Comment:
Did you test this change in the new minimal old environment?

Based on the CI failures, this does not work, and we must deploy the crds separately, and keep this wait before we deploy the olm yaml.

Unless we find another way, I think having 2 simple kustomization files and waiting until the crds are established before applying the second kustomization is the way.

Something like:

    crds/
       kustomization.yaml
    operators/
       kustomization.yaml

And keep 2 caches (olm-crds, olm-operators), like cdi/kubevirt.

Reply:
What do you mean by "after"? running the addon after drenv start ran?

Running the addon again on a system which you already deployed it does nothing, and cannot fail since the CRDs are already in the cluster. The error we see is trying to deploy the operator before the CRDs are established.